### DISCUSSION TITLE
deleted wallet

### DISCUSSION THREAD

**Discussion by mohoksoft** [btcpayserver/btcpayserver#3970] [Technical Support]
I installed btc payserver on my computer and started it. It synced and I created a wallet and store and deposited money into the wallet. I deleted my wallet by mistake. Is there a possibility to recover the money from the deleted wallet. Note that I am the owner of the server.
Is there a way to access that data through the database?

**Comment by pavlenex** [comment]
Did you back up your wallet? How did you delete it by mistake?

**Reply by mohoksoft** [reply]
No I am not back up my wallet. I am lost passphrase and try to replace wallet and on new wallet there is not transferred funds from old wallet. Then I deleted new wallet thinking I'll back to old but there is not old wallet and not funds.

**Reply by pavlenex** [reply]
So you've created a new wallet in BTCPay, a new hot wallet and set up a passphrase which you forgot. Then you went and replaced that wallet despite the warning, you typed in `DELETE` and replaced it am I correct on all steps?
![Screenshot 2022-07-14 at 12 50 28](https://user-images.githubusercontent.com/36959754/178966152-068a7390-314d-4b4c-8a1b-f439dd3e4225.png)

In that case, your funds are lost and cannot be recovered.

**Reply by mohoksoft** [reply]
First I typed replace and setup new wallet.

**Reply by pavlenex** [reply]
Correct, well as the warrning says, your wallet data is wiped from the server. I cannot think of a way to recover it, perhaps @Kukks may have some crazy idea, but most likely it's a dead end. In any case ,if you with some miracle mange to recover it, you wont' be able to use it in case you forgot the passphrase.

**Reply by mohoksoft** [reply]
But thank you for fast respond. It's not a huge amount of money, around $300, but I could drink it with my friends.

**Reply by Kukks** [reply]
Do you still have the xpub of the original wallet? @NicolasDorier  we store the key inside nbx, so  the data is still there

**Reply by mohoksoft** [reply]
No, I don't have an xpub original wallet, but I can access the database, if that would help.
